packages/react-ui-components/package.json

```diff
@@ -62,5 +62,8 @@
   },
   "jest": {
     "preset": "@neos-project/jest-preset-neos-ui"
+  },
+  "dependencies": {
+    "cross-env": "^5.1.3"
```

Comment:
Could you explain why you are adding this dependency here?

Reply:
Yes I can ... it was to late and I found cross-env in the package.json of react-ui-components... but missed that it was not in dependencies :(